.github/ISSUE_TEMPLATE/feature_request.yml

```diff
@@ -1,49 +1,56 @@
 name: "⚡ Feature Request"
 description: Suggest a new feature
 title: "[FEATURE]: <Title>"
-labels: ["kind/feature ", "status/triage"]
+labels: ["kind/feature"]
```

Comment:
the nice part of keeping `"status/triage"` is that we know what we have NOT triaged.